client/blocks/inline-help/popover.jsx

```diff
@@ -205,17 +235,51 @@ class InlineHelpPopover extends Component {
 		);
 	};
 
+	checkForClassicEditorOnAtomic() {
+		const { siteId, sitePlugins, showErrorNotice, translate } = this.props;
+		const classicPlugin = find( sitePlugins, { slug: 'classic-editor' } );
+
+		if ( ! classicPlugin ) {
+			showErrorNotice(
+				translate(
+					'There was a problem activating the Classic editor on your site. Please contact support.'
+				)
+			);
+			return;
+		}
+
+		if ( ! classicPlugin.active ) {
+			this.setState( { activatingClassicOnAtomic: true } );
+			this.props.activatePlugin( siteId, classicPlugin );
+			return;
+		}
+
+		this.redirectToClassicEditor();
+	}
+
 	switchToClassicEditor = () => {
-		const { siteId, onClose, optOut, classicUrl, translate } = this.props;
+		const { siteId, translate, isAtomic, sitePlugins } = this.props;
+
 		const proceed =
 			typeof window === 'undefined' ||
 			window.confirm( translate( 'Are you sure you wish to leave this page?' ) );
+
 		if ( proceed ) {
-			optOut( siteId, classicUrl );
-			onClose();
+			if ( isAtomic && isEnabled( 'editor/after-deprecation' ) ) {
+				this.checkForClassicEditorOnAtomic( siteId, sitePlugins );
```

Comment:
No need to pass any arguments to `checkForClassicEditorOnAtomic`.

Reply:
woops, forgot to tidy that up - thanks.